Makefile

```diff
@@ -366,9 +366,9 @@ venv:
 setup: venv
 	$(MAKE) -C tools
 	$(PYTHON) tools/decompress_baserom.py $(VERSION)
-# TODO: for now, we only extract ROM segments and assets from the Debug ROM
+	$(PYTHON) tools/extract_baserom.py $(BASEROM_DIR)/baserom-decompressed.z64 -o $(BASEROM_SEGMENTS_DIR) --dma-start `cat $(BASEROM_DIR)/dma_start.txt` --dma-names $(BASEROM_DIR)/dma_names.txt
```

Comment:
Can you change the file names to be `dmadata_` instead of `dma_`, not that there's a risk of confusion but [DMA](https://en.wikipedia.org/wiki/Direct_memory_access) != dmadata (what the og devs called the table and how we refer to it in the repo as well) and I like technical correctness

fwiw the `--dma-start` in compress.py is because it was `--dma-range` in my script, which I mimicked from z64compress' command line argument `--dma`
`--dmadata-start` would be better

I guess you can also apply this comment to `--dma-` args in extract_baserom.py

Reply:
Okay, I tried to change everything in sight to use the new names too